src/PointwiseFunctions/GeneralRelativity/Python/Psi4Real.cpp

```diff
@@ -0,0 +1,41 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/GeneralRelativity/Python/Psi4Real.hpp"
+
+#include <cstddef>
+#include <pybind11/pybind11.h>
+#include <string>
+#include <type_traits>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Psi4Real.hpp"
+
+namespace py = pybind11;
+
+namespace GeneralRelativity::py_bindings {
+
+namespace {
+template <typename Frame>
+void bind_psi4real_impl(py::module& m) {  // NOLINT
+  m.def(
```

Comment:
you can put bindings of pointwise functions directly in the `Bindings.cpp`, that reduces the number of boilerplate / files considerably.